RELEASE_NOTES

```diff
@@ -6,6 +6,7 @@
   - add a TemplateEngine based on java.text.MessageFormat
   - allow absence of 'value' on @MaxRows and validate its (lack of) use on SqlObjects
   - fix SqlObject getting confused about result types due to bridge methods
+  - StringTemplate no longer loads templates concurrently to prevent races
```

Comment:
With the change I just merged into this branch, this should probably read "StringTemplate no longer shares template group across threads, to avoid concurrency problems withing STGroup" or something like that.